docs/mkdocs/documentation/kmod_image.md

```diff
@@ -141,7 +141,7 @@ COPY --from=builder /usr/src/kernel-module-management/ci/kmm-kmod/kmm_ci_a.ko /o
 COPY --from=builder /usr/src/kernel-module-management/ci/kmm-kmod/kmm_ci_b.ko /opt/lib/modules/${KERNEL_FULL_VERSION}/
 
 # Create the symbolic link
-RUN ln -s /lib/modules/${KERNEL_FULL_VERSION} /opt/lib/modules/${KERNEL_FULL_VERSION}/host
+RUN ln -s /usr/lib/modules/${KERNEL_FULL_VERSION} /opt/lib/modules/${KERNEL_FULL_VERSION}/host
```

Comment:
I suspect this may not work in diestros in which `/lib` isn't linked to `/usr/lib`.

AFAIK, `/lib/modules` will always work and if there is an internal linkage to something else then it still work.

@yevgeny-shnaidman WDYT about this?

Reply:
since this is an example i also don't think this line should be changed.  /lib/module is the conventional place , in most of the systems it is linked to /usr/lib/modules, and in those that those directories are separate - then most of the drivers' files will be present in the /lb/modules